File: Codes/Dalily_Flash/Week_3/31Jan/Java/Program5.java
import java.util.*;

class PerfectNumber {

	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);
		System.out.print("Enter Limit = ");
		int limit = sc.nextInt();
		int temp;
		for(int no = 1;no<=limit;no++) {
			
			temp=0;
			for(int no1=1;no1<no;no1++) {

				if(no%no1==0) {

					temp = temp+no1;
				}
			}

			if(temp==no) {

				System.out.print(no+" ");
			}
		}
		System.out.println();

	}
}

File: Codes/Dalily_Flash/Week_3/31Jan/Python/Program5.py
lim = int(input("Enter Limit = "))

for no in range(1,lim):
    temp=0
    for no1 in range(1,no):
        if(no%no1==0):
            temp = temp+no1
    if(no==temp):
        print(no,end=" ")
print()
